commit ecad6c28195e535ae8ea8a29a91fa14f5e39faa9
Author: Andrzej Telszewski <andrzej@telszewski.com>
Date:   Wed Apr 7 22:55:57 2021 +0200

    Fix build error with Qt 5.15
    
    Shamelessly based on upstream patch:
      https://github.com/wereturtle/ghostwriter/commit/f8f4ec0a1b53d1f3e670817f051c59444851cf7f

diff --git a/src/MarkdownEditor.cpp b/src/MarkdownEditor.cpp
index 2e4a9a1..45fc3e4 100644
--- a/src/MarkdownEditor.cpp
+++ b/src/MarkdownEditor.cpp
@@ -34,6 +34,7 @@
 #include <QUrl>
 #include <QPixmap>
 #include <QPainter>
+#include <QPainterPath>
 #include <QFileInfo>
 #include <QDir>
 
